src/projectConfig.js

```diff
@@ -44,6 +45,16 @@ export default {
         uniqueItems: true,
       }],
     },
+    sync: {
+      type: 'object',
+      properties: {
+        hooks: {
```

Comment:
```suggestion
        hooks: { $ref: "#/definitions/require }
```
? or maybe we'd make a separate `hooks` definition for it?

Reply:
The definition of require should work well for this! If I made a separate one for hooks, would essentially be the same thing.